dspy/predict/predict.py

```diff
@@ -50,6 +52,9 @@ def dump_state(self, save_verbose=None):
                 if isinstance(demo[field], BaseModel):
                     demo[field] = demo[field].model_dump_json()
 
+                if is_image(demo[field]):
```

Comment:
TODO: Test this

Reply:
have we tested this

Reply:
in any case, idk if this belongs in this file, shouldn't it be handled in the adapter or in the signature or something?

Reply:
I did not test this rip